### ISSUE TITLE
removal of utils.h breaks building for Windows / mingw

### ISSUE THREAD

**Issue by kaie** [rnpgp/rnp#960]
Cross-compiling RNP for Windows on Linux fails for src/rnp/rnp.cpp because it causes the wrong rexec code to be used.

The following commit removed the include header "utils.h", which caused the build to break:
https://github.com/rnpgp/rnp/commit/b4f58ff0c7055f7798b243abfb80d3fcb2473946#diff-ffe9c76b36d264aaf899a0fe7633c45c

Do you need a pull request from me, or could you simply add it back as part of some other planned commit?

```
diff --git a/src/rnp/rnp.cpp b/src/rnp/rnp.cpp
index daf4f9f3..c7dcd935 100644
--- a/src/rnp/rnp.cpp
+++ b/src/rnp/rnp.cpp
@@ -48,6 +48,8 @@
 #include "fficli.h"
 #include "rnpcfg.h"
 
+#include "utils.h"
+
 // must be placed after include "utils.h"
 #ifndef RNP_USE_STD_REGEX
 #include <regex.h>
```

**Comment by ni4** [comment]
@kaie yeah, that's my fault. I addressed this in (yet draft) PR #952 . It also provides fixes required to run rnp_tests ( and those run successfully). cli_tests do not work yet though. And another part of that PR is CI within GHA for Windows, which still work in progress.

**Comment by ronaldtse** [comment]
@kaie and that's why we're also enabling GHA for Windows, will get that running soon. We've got a number of new folks here that will help ramp up the Windows side of things.

**Comment by ni4** [comment]
Fixed in #952
